openedx/features/colaraz_features/api/views.py

```diff
@@ -80,3 +80,45 @@ def get(self, request, api_method, format=None):
             {'message': 'Notifications API is not configured properly'},
             status=status.HTTP_400_BAD_REQUEST
         )
+
+
+class JobAlertsHandlerApiView(APIView):
+    """
+    APIView to fetch job alerts and mark them as read.
+    """
+    authentication_classes = (SessionAuthentication,)
+    permission_classes = (IsAuthenticated,)
+
+    API_ENDPOINTS = {
+        'fetch': 'FETCH_URL',
+        'mark': 'MARK_URL'
+    }
+
+    def get(self, request, api_method, format=None):
+        """
+        This method uses Colaraz's Job Alerts API to fetch and mark alerts
+        and returns data sent by API in json format.
```

Comment:
Just a thought. 
can we add a response format in the docs too?

Reply:
I think we can, as its a good idea. But is edX following this pattern?

Reply:
I have seen it in some places if not everywhere.

Reply:
Added.